src/gettingstarted/own-code/service-configuration.md

```diff
@@ -0,0 +1,69 @@
+# Import your own code
+
+## Configure services
+
+In the previous step, you created a collection of empty configuration files that have given your project the following structure:
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+Now you will need to include information that will tell Platform.sh how you want your application to connect to its [services](/configuration/services.md). An example `.platform/services.yaml` will always look something like this:
+
+`.platform/services.yaml`
+
+{%- codetabs name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform/services.yaml" -%}
```

Comment:
This doesn't need to vary by language, and several of these are empty anyway.  Just custom-write something and inline it, no tabs needed.